<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TK_BPS_ACC</string>
  </property>
  <widget class="QTableWidget" name="tb_down_data">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>231</height>
    </rect>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>8</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>c0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c7</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>通道</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ADC值</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>偏移值</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>比例系数值</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ADC端口电压</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>输入总压(mV)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>输入单体(mV)</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tb_ac_data">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>321</width>
     <height>231</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>c0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c7</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>输入电压A</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>输入电压B</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AC_offset</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AC_rate</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>3288</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string> </string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>6580</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string> </string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>9850</string>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string> </string>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string>13160</string>
    </property>
   </item>
   <item row="3" column="1">
    <property name="text">
     <string> </string>
    </property>
   </item>
   <item row="3" column="2">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string>16460</string>
    </property>
   </item>
   <item row="4" column="1">
    <property name="text">
     <string> </string>
    </property>
   </item>
   <item row="4" column="2">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="5" column="0">
    <property name="text">
     <string>19660</string>
    </property>
   </item>
   <item row="5" column="1">
    <property name="text">
     <string> </string>
    </property>
   </item>
   <item row="5" column="2">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="6" column="0">
    <property name="text">
     <string>22960</string>
    </property>
   </item>
   <item row="6" column="1">
    <property name="text">
     <string> </string>
    </property>
   </item>
   <item row="6" column="2">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="7" column="0">
    <property name="text">
     <string>22840</string>
    </property>
   </item>
   <item row="7" column="1">
    <property name="text">
     <string> </string>
    </property>
   </item>
   <item row="7" column="2">
    <property name="text">
     <string>0</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_choice_data">
   <property name="geometry">
    <rect>
     <x>541</x>
     <y>251</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>一次校准</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_acc">
   <property name="geometry">
    <rect>
     <x>541</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>二次校准</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tb_Vo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>191</width>
     <height>231</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>c0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>c7</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>输出电压A</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>输出电压B</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btn_set_acc">
   <property name="geometry">
    <rect>
     <x>541</x>
     <y>309</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>标定</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>430</y>
     <width>62</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lb_id">
      <property name="text">
       <string>ID: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_fbnum">
      <property name="text">
       <string>fb_num: 00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_conn">
      <property name="text">
       <string>connect..</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btn_choice_data</tabstop>
  <tabstop>btn_acc</tabstop>
  <tabstop>btn_set_acc</tabstop>
  <tabstop>tb_ac_data</tabstop>
  <tabstop>tb_Vo</tabstop>
  <tabstop>tb_down_data</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
